hibernate-reactive-core/src/main/java/org/hibernate/reactive/engine/impl/ReactiveEntityUpdateAction.java

```diff
@@ -141,12 +141,11 @@ private CompletionStage<EntityEntry> handleGeneratedProperties(EntityEntry entry
 		}
 	}
 
-	// TODO: copy/paste from superclass (make it protected)
-	private void handleDeleted(EntityEntry entry, EntityPersister persister, Object instance) {
+	protected void handleDeleted(EntityEntry entry, EntityPersister persister, Object instance) {
```

Comment:
The comment is not about making this method protected, it's about making the method in ORM protected so that we can remove this code from Reactive